src/notation/internal/excerptnotation.cpp

```diff
@@ -133,7 +133,9 @@ void ExcerptNotation::setName(const QString& name)
 
     excerptTitle->setPlainText(name);
     score()->setMetaTag(u"partName", name);
-    score()->doLayout();
+    if (score()->isOpen()) {
+        score()->doLayout();
```

Comment:
I think this call can be completely removed from here

Reply:
done, thanks!